src/components/Drawer/utils.tsx

```diff
@@ -0,0 +1,10 @@
+import { Dispatch, SetStateAction } from 'react';
+
+export const toggleDrawer =
+  (open: boolean, setOpen: Dispatch<SetStateAction<boolean>>) => (event: any) => {
+    if (event && event.type === 'keydown' && (event.key === 'Tab' || event.key === 'Shift')) {
```

Comment:
I see that the `toggleDrawer` util is only used for the mobile case. Does it make sense to check for `keydown` and `key === 'Tab'/'Shift'` events?

Reply:
That's correct, but what if the window on the desktop is small? With this implementation it will handle this use case.

Reply:
I see! It's good then!